extensions/blocks/conversation/view.scss

```diff
@@ -0,0 +1,12 @@
+/*
+ * Conversation / Dialogue block styles
+ */
+ .wp-block-jetpack-conversation {
+	.wp-block-jetpack-dialogue__timestamp {
+		display: none;
+	}
+
+	&.show-timestamp .wp-block-jetpack-dialogue__timestamp {
+		display: block;
+	}
+ }
```

Comment:
Let's move this to `style.scss` maybe, and get rid of `view.scss` here and in the reference in `extensions/blocks/conversation/view.js`?